### ISSUE TITLE
Poll: Do you use a host other than the default ("0.0.0.0")?

### ISSUE THREAD

**Issue by natekspencer** [briis/smartweatherudp#37]
[![](https://api.gh-polls.com/poll/01FP0TFX903DP9WNCN1J18RR9Y/Yes)](https://api.gh-polls.com/poll/01FP0TFX903DP9WNCN1J18RR9Y/Yes/vote)
[![](https://api.gh-polls.com/poll/01FP0TFX903DP9WNCN1J18RR9Y/No)](https://api.gh-polls.com/poll/01FP0TFX903DP9WNCN1J18RR9Y/No/vote)

**Comment by chheiss** [comment]
No, that is not working for me. Only 0.0.0.0 works. Direct host IP shows no connection, and host on another subnet is not recognized either.
Unfortunately!
